drivers/video/mt9m114.c

```diff
@@ -53,6 +55,10 @@ LOG_MODULE_REGISTER(video_mt9m114, CONFIG_VIDEO_LOG_LEVEL);
 #define MT9M114_CAM_OUTPUT_FORMAT_FORMAT_YUV (0 << 8)
 #define MT9M114_CAM_OUTPUT_FORMAT_FORMAT_RGB (1 << 8)
 
+/* Camera control masks */
+#define MT9M114_CAM_CONTROL_HORIZ_FLIP_EN    BIT(0)
+#define MT9M114_CAM_CONTROL_VERT_FLIP_EN     BIT(1)
```

Comment:
I suggest : `MT9M114_CAM_SENSOR_CTRL_VERT_FLIP_EN` to be coherent with `MT9M114_CAM_SENSOR_CTRL_READ_MODE`